queries.sql

```diff
@@ -8,4 +8,6 @@ SELECT * FROM animals WHERE neutered = TRUE AND escape_attempts < 3;
 /* List the date of birth of all animals named either "Agumon" or "Pikachu"*/
 SELECT * FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
 /*List name and escape attempts of animals that weigh more than 10.5kg*/
-SELECT name FROM animals WHERE weight_kg > 10;
\ No newline at end of file
+SELECT name FROM animals WHERE weight_kg > 10;
```

Comment:
- [ ] The weight should be greater than 10.5 and not 10. Kindly rectify this issue 😄